WORKSPACE

```diff
@@ -5,6 +5,10 @@ load("//format:repositories.bzl", "rules_format_dependencies")
 # Fetch dependencies which users need as well
 rules_format_dependencies()
 
+load("//format:dependencies.bzl", "rules_format_setup")
```

Comment:
I think it should be possible to make this PR without changing this file, since that's pretty annoying for users to have to account for it. Can you put your logic in `format_register_toolchains` ?

Reply:
Ahh, didn't notice there's already a place for it